src/config/server.ts

```diff
@@ -158,7 +158,8 @@ const SERVER_CONFIG: StrictServerConfiguration = {
     timestampCacheFix: true,
     useAjvCycleRecordValidation: true,
     networkTransactionsToProcessPerCycle: 20,
-    getTxTimestampTimeoutOffset: 0
+    getTxTimestampTimeoutOffset: 0,
+    timestampCacheFixsize: 10000
```

Comment:
can you fix the casing of this.  Size should have an uppercase 'S'

Reply:
I have fixed the following issue.